src/postgres-util/src/desc.rs

```diff
@@ -135,18 +265,74 @@ impl Arbitrary for PostgresColumnDesc {
             any::<u32>(),
             any::<i32>(),
             any::<bool>(),
-            any::<bool>(),
         )
             .prop_map(
-                |(name, col_num, type_oid, type_mod, nullable, primary_key)| PostgresColumnDesc {
+                |(name, col_num, type_oid, type_mod, nullable)| PostgresColumnDesc {
                     name,
                     col_num: Some(col_num),
                     type_oid,
                     type_mod,
                     nullable,
-                    primary_key,
                 },
             )
             .boxed()
     }
 }
+
+/// Describes a kye in a [`PostgresTableDesc`].
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, PartialOrd, Ord)]
+pub struct PostgresKeyDesc {
+    /// This key is derived from the `pg_constraint` with this OID.
+    pub oid: u32,
+    /// The name of the column.
+    pub name: String,
+    /// The `attnum` of the columns comprising the key.
+    pub cols: Vec<u16>,
```

Comment:
Should this be named `attnum`? Also for the purposes of readers not very familiar to pg internals, is there an even more descriptive name/doc comment?